### PR TITLE
226 - [Product] Implement Order tab with tables

### LINKED ISSUE #226: [Product] Implement "Order" tab with tables
Implement & style the product order tab using "table" block - this requires #173 

Samples with all in one table
* https://main--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers#order
* https://main--moleculardevices--hlxsites.hlx.page/products/flipr-penta-high-throughput-cellular-screening-system#order
* https://main--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers#order

Samples with multiple tables
* https://main--moleculardevices--hlxsites.hlx.page/products/axon-patch-clamp-system/digitizers/axon-digidata-1550b-plus-humsilencer#order

Samples with 3 columns
* https://main--moleculardevices--hlxsites.hlx.page/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software#order

### PR DESCRIPTION
**Pull request by andreituicu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #226

H2s on the Order tab are not styled, as they are part of https://github.com/hlxsites/moleculardevices/pull/259.

Test URLs:

- Original: https://moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers
- After: https://226-order-table--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers

- Original: https://moleculardevices.com/products/flipr-penta-high-throughput-cellular-screening-system
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/flipr-penta-high-throughput-cellular-screening-system
- After: https://226-order-table--moleculardevices--hlxsites.hlx.live/products/flipr-penta-high-throughput-cellular-screening-system

- Original: https://moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers
- After: https://226-order-table--moleculardevices--hlxsites.hlx.live/products/microplate-readers/multi-mode-readers/spectramax-i3x-readers

- Original: https://moleculardevices.com/products/axon-patch-clamp-system/digitizers/axon-digidata-1550b-plus-humsilencer
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/axon-patch-clamp-system/digitizers/axon-digidata-1550b-plus-humsilencer
- After: https://226-order-table--moleculardevices--hlxsites.hlx.live/products/axon-patch-clamp-system/digitizers/axon-digidata-1550b-plus-humsilencer

- Original: https://moleculardevices.com/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software
- Before: https://main--moleculardevices--hlxsites.hlx.live/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software
- After: https://226-order-table--moleculardevices--hlxsites.hlx.live/products/microplate-readers/acquisition-and-analysis-software/softmax-pro-software

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -410,8 +410,8 @@ async function loadLazy(doc) {
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
-  const megaMenuModule = await import('../blocks/header/header-megamenu.js');
-  megaMenuModule.default(headerBlock);
+  // const megaMenuModule = await import('../blocks/header/header-megamenu.js');
```
Comment (lydiapuric) [inline scripts/scripts.js:413]: The mega menu is commented, intended?

**Comment by lydiapuric** [comment]
Code looks good.
However, the styling in Mobile is not addressed and the font-sizes are too big.

**Comment by paolomoz** [comment]
@andreituicu 

- like @lydiapuric commented: megamenue should probably be uncommented right?
- on mobile: font size is 14px, should be 18px 
- border left /right of the table should be removed

**Comment by andreituicu** [comment]
> border left /right of the table should be removed

@paolomoz for me there are no left / right borders. how does it look for you? maybe I missed something.

Screenshot 2023-05-05 at 12 02 24

Screenshot 2023-05-05 at 12 02 39

**Comment by andreituicu** [comment]
> However, the styling in Mobile is not addressed and the font-sizes are too big.

> on mobile: font size is 14px, should be 18px

Do you have some example screenshots please? For me they look the same, but I'm for sure not looking in the right place.

Original Mobile: (the small Products in the header looks like a bug to me)

Screenshot 2023-05-05 at 12 10 34

Implementation in this PR:

Screenshot 2023-05-05 at 12 10 34

**Comment by lydiapuric** [comment]
Mobile looks good, and not reproducible anymore.

**Review by lydiapuric — APPROVED** [review]
+1 LGTM

**Review by mhaack — APPROVED** [review]

**Review by paolomoz — APPROVED** [review]

**Pull request merged**